### ISSUE TITLE
[bug] toRight and toLeft lose user scrolling

### ISSUE THREAD

**Issue by FezVrasta** [ef4/liquid-fire#373]
I'm showing a list of items, when user click on one of the, the list moves toLeft and a detail page appears from the right edge of the page.

Then, when user click on the back button, the detail page moves toRight and the list comes back from left edge at the same scrolling position of when the details page opened.

Those pages are loaded inside a {{liquid-outlet}}

The problem is that using toLeft and toRight automatically scroll the page to the top. I would expect the transition to not touch the page scrolling position.

**Comment by ConorLinehan** [comment]
Hey @FezVrasta How are you transitioning back a page??

**Comment by FezVrasta** [comment]
Using a link-to

**Comment by ConorLinehan** [comment]
I don't think that `{{link-to}}` or `this.transitionTo` have the functionality of retaining scroll position http://emberjs.jsbin.com/nihihocevu/edit?html,js,output you could try passing the model and scrolling to it based on div using jQuery, or you could use `window.history.back()` ?

**Comment by FezVrasta** [comment]
I already listen to the scroll event and I store it in the application controller. But when the liquid fire animation stars, the page automatically scrolls to the top. This is the problem.

**Comment by FezVrasta** [comment]
I mean, it's even ugly. When the animation changes, I see the page that scrolls super fastly to the top, and then slides on the side.
The animation instead should be just the side sliding

**Comment by ef4** [comment]
It sounds like you are trying to animate the whole viewport. That can be done, but it will probably require you to apply your own CSS to the liquid-container to get the scrolling behavior you want.

By default liquid-fire doesn't dictate anything about the heights of your content. But you will need to fix some heights to keep the page scrollHeight stable.
